lib/db_helper/tasks.rb

```diff
@@ -12,7 +12,8 @@
   desc "Copy and import a remote database to a local database"
   task :import_from_remote do
     begin
-      sources = YAML::load(File.open(File.join(ENV['RAILS_ROOT'] || '.', File.join('config','database_import.yml'))))
+      path = File.join(ENV['RAILS_ROOT'] || '.', 'config', 'db_helper_import_from_remote.yml')
```

Comment:
Line is too long. [94/80]<br>Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.